charts/linkerd-control-plane/templates/destination.yaml

```diff
@@ -264,6 +264,7 @@ spec:
         - --default-policy={{.Values.policyController.defaultAllowPolicy}}
         - --log-level={{.Values.policyController.logLevel | default "linkerd=info,warn"}}
         - --log-format={{.Values.controllerLogFormat}}
+        - --probe-networks={{.Values.policyController.probeNetworks | join ","}}
```

Comment:
Can we expect people to use an empty list at any point here? e.g `linkerd install --set policyController.probeNetworks=[]` would theoretically be valid as a template value. In this case, however, the policy-controller would fail with:

```
error: Invalid value "[]" for '--probe-networks <PROBE_NETWORKS>': invalid IP address syntax
```

I'm not sure how likely it is for someone to override the value to be an empty array, but should we deal with this case by checking the array is not empty (and if it is, fail)?